src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,4 +1,11 @@
-// Add the required props
-export const UserInfo = () => (
-  <>UserInfo markup</>
-);
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => {
+  const { email, username } = user;
```

Comment:
There is no null check for 'user' prop. This can lead to a potential error if the 'user' object is undefined. Add a null check before destructuring the 'user' object.